### ISSUE TITLE
improper JSON in code example

### ISSUE THREAD

**Issue by rendall** [MicrosoftDocs/minecraft-creator#732]
This is the code example given in on the page:

```JSON
{
  "format_version": "1.19.40",
  "minecraft:block": {
    "description": {
      "identifier": "custom:lamp_block",
      "state": {
        "custom:is_lit": [ true, false ]
      }
    },
    "permutations": {
      {
        "condition": "query.block_state('custom:is_lit') == true",
        "components": {
          "minecraft:light_emission": 15
        }
      },
      {
        "condition": "query.block_state('custom:is_lit') == false",
        "components": {
          "minecraft:light_emission": 0
        }
      }
    },
    "components": {
      "minecraft:friction": 0.6,
      "minecraft:map_color": "#00ff00" 
    }
  }
}
```

However, this is not properly formatted JSON, since the value of "permutations" uses curly brackets and not square brackets but contains objects without keys, like an array.

This is probably correct:

```JSON
{
  "format_version": "1.19.40",
  "minecraft:block": {
    "description": {
      "identifier": "custom:lamp_block",
      "state": {
        "custom:is_lit": [ true, false ]
      }
    },
    "permutations": [ 
      {
        "condition": "query.block_state('custom:is_lit') == true",
        "components": {
          "minecraft:light_emission": 15
        }
      },
      {
        "condition": "query.block_state('custom:is_lit') == false",
        "components": {
          "minecraft:light_emission": 0
        }
      }
     ],
    "components": {
      "minecraft:friction": 0.6,
      "minecraft:map_color": "#00ff00" 
    }
  }
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 569e663b-eab4-840d-00d6-3393bee89d2f
* Version Independent ID: 569e663b-eab4-840d-00d6-3393bee89d2f
* Content: [Block Documentation - Block States and Permutations](https://learn.microsoft.com/en-us/minecraft/creator/reference/content/blockreference/examples/blockstatesandpermutations)
* Content Source: [creator/Reference/Content/BlockReference/Examples/BlockStatesAndPermutations.md](https://github.com/MicrosoftDocs/minecraft-creator/blob/main/creator/Reference/Content/BlockReference/Examples/BlockStatesAndPermutations.md)
* Product: **gaming**
* GitHub Login: @JimSeaman42
* Microsoft Alias: **mikeam**

**Comment by rendall** [comment]
Additionally, `"state"` may be incorrect.  I have not verified yet, but according to <https://learn.microsoft.com/en-us/minecraft/creator/reference/content/blockreference/examples/blockdescription> the field is "states" (plural).

**Comment by rendall** [comment]
I have verified that `"states"` (plural) is the proper field and not `"state"`.

This is the corrected JSON:

```JSON
{
  "format_version": "1.19.40",
  "minecraft:block": {
    "description": {
      "identifier": "custom:lamp_block",
      "states": {
        "custom:is_lit": [ true, false ]
      }
    },
    "permutations": [ 
      {
        "condition": "query.block_state('custom:is_lit') == true",
        "components": {
          "minecraft:light_emission": 15
        }
      },
      {
        "condition": "query.block_state('custom:is_lit') == false",
        "components": {
          "minecraft:light_emission": 0
        }
      }
     ],
    "components": {
      "minecraft:friction": 0.6,
      "minecraft:map_color": "#00ff00" 
    }
  }
}
```

Furthermore, unless the "Holiday Creator Experiment" is enabled, accessing the custom state will error.

**Comment by rendall** [comment]
Addressed this issue with PR #733
